mocks/hit-endpoints.js

```diff
@@ -0,0 +1,99 @@
+const axios = require('axios').default;
+
+const settings = require('./settings');
+
+sendRequestToAllEndpoints();
+
+async function sendRequestToAllEndpoints() {
+  const routeResponse = await axios.get(
+    'http://localhost:3110/api/mock/routes'
+  );
+
+  for (let i = 0; i < routeResponse.data.length - 1; i++) {
+    const entry = routeResponse.data[i];
+
+    if (entry.id in CUSTOM_REQUESTS) {
+      CUSTOM_REQUESTS[entry.id]();
+      continue;
+    }
+
+    // Regex to replace all url parameters, like for example '/:something/' becomes '/1/'.
+    const paramReplacingPattern = new RegExp('(?<=\/):.+?(?=\/|$)', 'g'); // prettier-ignore
+    // reason: It will remove all escape sequences in the regular expression which is undesired.
+
+    const requestUrl = `${settings.MOCK_ORIGIN}${entry.url.replaceAll(paramReplacingPattern, '1')}`;
+
+    if (entry.method === 'get') {
+      axios.get(requestUrl);
+    } else if (entry.method === 'post') {
+      axios.post(requestUrl);
+    }
```

Comment:
maybe a console.debug for the not implemented entry methods?

Reply:
Goed idee, ik heb er een error van gemaakt als je dat goed vind. Omdat het een configuratie fout van de ontwikkelaar is die ik vind meteen gefixed dient te worden. En daardoor zag ik ook meteen een route die niet goed geconfigureerd is.

Reply:
ja true!